nfs-ganesha/build/build_rpm

```diff
@@ -117,12 +118,12 @@ if [ "$THROWAWAY" = false ] ; then
 }
 EOF
     # post the json to repo-extra json to chacra
-    curl -X POST -H "Content-Type:application/json" --data "@$WORKSPACE/repo-extra.json" -u $CHACRACTL_USER:$CHACRACTL_KEY ${chacra_url}/repos/${chacra_endpoint}/extra/
+    curl -X POST -H "Content-Type:application/json" --data "@$WORKSPACE/repo-extra.json" -u $CHACRACTL_USER:$CHACRACTL_KEY ${chacra_url}repos/${chacra_repo_endpoint}/extra/
```

Comment:
It depends on how you've defined ``$chacra_url``, did you include a trailing slash?